src/oidcendpoint/endpoint_context.py

```diff
@@ -144,6 +136,7 @@ def __init__(
         self.login_hint2acrs = None
         self.userinfo = None
         self.scope2claims = SCOPE2CLAIMS
+        self.token_usage_rules = {}
```

Comment:
Is there any way to initialize this through configuration, on initialization? 

I can't find out how to define this other that setting after the endpoint_context has been initialized:
```
ctx = EndpointContext()
ctx.token_usage_rules = some_dict
```

Reply:
Ah, there is a disconnect here. I'll fix that and will add a couple of tests.
You can find token_usage_rules in the configuration of test_35_... and test_50_... but not used :-(

Reply:
Hmm, have been a bit unclear in my thinking here. Token usage rules are part of the authorization handling so it should not be on the top-level of endpoint_context. Look at test_35_.. authz/kwargs/grant_config/ that is where it should be.